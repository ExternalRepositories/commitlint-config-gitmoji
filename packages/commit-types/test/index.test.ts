import types from '../src';

test('check types', async () => {
  expect(types).toMatchSnapshot();
});
