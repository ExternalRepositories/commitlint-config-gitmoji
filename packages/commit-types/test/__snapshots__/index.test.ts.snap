// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check types 1`] = `
Array [
  "build",
  "ci",
  "docs",
  "feat",
  "fix",
  "perf",
  "refactor",
  "revert",
  "style",
  "test",
  "chore",
  "wip",
]
`;
